src/style.css

```diff
@@ -1,3 +1,69 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  --main-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto , sans-serif;
+  font-size: 12px;
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.nav-list {
+  display: flex;
+  list-style-type: none;
+}
+
+.header-nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+}
+
+.nav-list__item {
+  margin-left: 20px;
+}
+
+.nav-list__item:first-child {
+  margin-left: 0;
+}
+
+.nav-list__link {
+  display: inline-block;
+  height: 60px;
```

Comment:
It's okay in this case, but it's better to avoid hardcoding width/height, you can define your container size with paddings or line-height in your solution.